README.md

```diff
@@ -18,11 +18,22 @@ CleanVision is super simple -- run the same couple lines of Python code to audit
 [![Cleanlab Studio](https://raw.githubusercontent.com/cleanlab/assets/master/shields/cl-studio-shield.svg)](https://cleanlab.ai/studio/?utm_source=github&utm_medium=readme&utm_campaign=clostostudio)
 
 ## Installation
```

Comment:
Revert this to shorter form from before, move the mention of HF and torch datasets to more resources with the links to the notebooks